uber/site_sections/summary.py

```diff
@@ -175,12 +175,115 @@ def food_eligible(self, session):
         }
         return render('summary/food_eligible.xml', {'attendees': eligible})
 
+    def csv_import(self, message='', all_instances=None, form_link=None):
```

Comment:
btw for a long while had been thinking about adding some kind of 'superuser' and 'developer' access level to do things that potentially mess with the database on a larger scale (like importing stuff here, fixing up badge numbers, running checks, etc), this might be a good reason to add it.